tests/utils.go

```diff
@@ -2155,6 +2155,19 @@ func WaitForVMConditionRemovedOrFalse(virtClient kubecli.KubevirtClient, vm *v1.
 	}, time.Duration(timeoutSec)*time.Second, 2).Should(BeFalse(), fmt.Sprintf("Should have no or false %s condition", conditionType))
 }
 
+func WaitExportProxyReady() {
```

Comment:
I think that for the sake of not making the `utils.go` grow you can move this function to `tests/testsuite/fixture.go`

Reply:
Agreed! Will move it. Thanks